examples/algorithms/plot_greedy_coloring.py

```diff
@@ -94,4 +94,21 @@ def _frame_update(index):
     frames=100,
 )
 
+## To save the animation to a local file, comment in the following for GIF:
+# ani.save(
+#     "greedy_coloring.gif",
+#     writer="imagemagick",
+#     savefig_kwargs={"facecolor": "white"},
+#     fps=20,
+# )
+## Or/end the following for MP4 in AV1 coding:
+# Writer = animation.writers["ffmpeg"]
+# writer = Writer(
+#     fps=20,
+#     metadata=dict(artist="NetworkX"),
+#     bitrate=1800,
+#     extra_args=["-vcodec", "av1", "-strict", "experimental"],
+# )
+# ani.save("greedy_coloring.mp4", writer=writer)
```

Comment:
Hmm, this is under the assumption the user has `imagemagick`/`ffmpeg` available locally. Should we write something anything about that?

Reply:
IMO this is out-of-scope for the NX gallery - this is matplotlib functionality and I think users would be much better served by the matplotlib docs on this topic rather than us trying to repeat/maintain it here!

Reply:
I agree that the drawings in our gallery should not be focused on how to use matplotlib features. Obviously they can/will use the tools we provide that use matplotlib. But the focus should be on networkx functionality.  The PR to add 3D to greedy_coloring was perhaps a step over that line already. 
It differs from the 3D drawing example. Our 3D layout routines exist and may be of interest, so the 3D graph drawing example is nice to show that possibility. 

Perhaps you should put a shorter comment in the 3D drawing example pointing to: [the matplotlib functionality for imagemajick](https://matplotlib.org/stable/api/_as_gen/matplotlib.animation.ImageMagickWriter.html) and [the matplotlib functionality for ffmpeg](https://matplotlib.org/stable/api/_as_gen/matplotlib.animation.FFMpegWriter.html)

I would be -1 for putting commented out detailed code in our examples because it will probably not work within a few months.